content/getting-started/sections/installing-atom.md

```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
 
 #### Installing Atom on Linux
 
-To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. These packages do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process.
+To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. If you'd like Atom to be updated when new releases are published, you can configure an official package repository for your distribution.
 
 [releases]: https://github.com/atom/atom/releases/latest
 
 ##### Debian and Ubuntu (deb/apt)
 
-To install Atom on Debian, Ubuntu, or related systems:
+To install Atom on Debian, Ubuntu, or related distributions, add our official
+package repository to your system by running the following commands:
+
+```
+curl -L https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
+sudo sh -c 'echo "deb [arch=amd64] deb https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
+sudo apt-get update
```

Comment:
I believe it's now recommended to use `apt` as a more beginner-friendly alternative, at least on Ubuntu.  Same for `apt install`.

Reply:
Thanks for pointing that out!  I think `apt` may only be available on Ubuntu variants and not Debian, so it may be better to use `apt-get` since it's the most universal command.  I'll mention that `apt` can also be used